<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Window</class>
 <widget class="QWidget" name="Window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>889</width>
    <height>716</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="font">
   <font>
    <pointsize>15</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Window</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(202, 241, 255)</string>
  </property>
  <widget class="QLabel" name="intelligence">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>170</y>
     <width>106</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Intelligence:</string>
   </property>
  </widget>
  <widget class="QLabel" name="intelligence_number">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>170</y>
     <width>58</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>intamt</string>
   </property>
  </widget>
  <widget class="QPushButton" name="marketing">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>110</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(138, 226, 52);</string>
   </property>
   <property name="text">
    <string>Activate Marketing</string>
   </property>
  </widget>
  <widget class="QPushButton" name="upgradeapm">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>350</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(114, 159, 207);</string>
   </property>
   <property name="text">
    <string>Upgrade APM</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="debugtool">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>420</y>
     <width>175</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>Debugging Tools </string>
   </property>
  </widget>
  <widget class="QPushButton" name="makepencils_button">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>20</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu</family>
     <pointsize>11</pointsize>
     <weight>7</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(43, 134, 224);
font: 57 11pt &quot;Ubuntu&quot;;
color: rgb(0, 0, 0)</string>
   </property>
   <property name="text">
    <string>Make Penciles</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget1">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>290</y>
     <width>256</width>
     <height>27</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <layout class="QHBoxLayout" name="autopencilrate_layout" stretch="0,0,0">
    <item>
     <widget class="QLabel" name="autopencilrate_label1">
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Rate:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="autopencilrate_int">
      <property name="font">
       <font>
        <family>Ubuntu</family>
        <pointsize>12</pointsize>
        <weight>7</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 57 12pt &quot;Ubuntu&quot;;</string>
      </property>
      <property name="text">
       <string>x amount</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="autopencilrate_label2">
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>pencils/minute</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget2">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>250</y>
     <width>228</width>
     <height>27</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <layout class="QHBoxLayout" name="autopencil_layout">
    <item>
     <widget class="QLabel" name="autopencil_label">
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>AutoPencile2000:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="autopencil_int">
      <property name="font">
       <font>
        <family>Ubuntu</family>
        <pointsize>12</pointsize>
        <weight>7</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 57 12pt &quot;Ubuntu&quot;;</string>
      </property>
      <property name="text">
       <string>x amount</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="bankbalance_label">
   <property name="geometry">
    <rect>
     <x>498</x>
     <y>67</y>
     <width>120</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Bank Balance:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buyautopencil_button">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>350</y>
     <width>131</width>
     <height>42</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu</family>
     <pointsize>11</pointsize>
     <weight>7</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(245, 121, 0);
font: 57 11pt &quot;Ubuntu&quot;;
color: rgb(0, 0, 0)</string>
   </property>
   <property name="text">
    <string>Buy More APMS
$100.00</string>
   </property>
  </widget>
  <widget class="QLabel" name="bankbalance_int">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>70</y>
     <width>112</width>
     <height>19</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu</family>
     <pointsize>12</pointsize>
     <weight>7</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 57 12pt &quot;Ubuntu&quot;;</string>
   </property>
   <property name="text">
    <string>x dollar amount</string>
   </property>
  </widget>
  <widget class="QPushButton" name="debug_pencils">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>460</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(114, 159, 207);</string>
   </property>
   <property name="text">
    <string>Pencils</string>
   </property>
  </widget>
  <widget class="QPushButton" name="debug_balance">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>460</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(173, 127, 168);</string>
   </property>
   <property name="text">
    <string>Balance</string>
   </property>
  </widget>
  <widget class="QPushButton" name="debug_resource">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>460</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(233, 185, 110);</string>
   </property>
   <property name="text">
    <string>Wood and Graphite</string>
   </property>
  </widget>
  <widget class="QPushButton" name="incmarketing">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>110</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(114, 159, 207);</string>
   </property>
   <property name="text">
    <string>Buy More Marketing</string>
   </property>
  </widget>
  <widget class="QPushButton" name="saveBtn">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>450</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QPushButton" name="loadBtn">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>450</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Load</string>
   </property>
  </widget>
  <widget class="QPushButton" name="exitButton">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>510</y>
     <width>161</width>
     <height>32</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(244, 66, 66)</string>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </widget>
  <widget class="QPushButton" name="intelligence_debug_button">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>510</y>
     <width>110</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(221, 117, 148)
</string>
   </property>
   <property name="text">
    <string>Intelligence</string>
   </property>
  </widget>
  <widget class="QPushButton" name="marketing_debug_button">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>510</y>
     <width>101</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(138, 226, 52)</string>
   </property>
   <property name="text">
    <string>Marketing</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pencils_inventory_debug_button">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>510</y>
     <width>176</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(245, 121, 0);</string>
   </property>
   <property name="text">
    <string>Pencils in Inventory</string>
   </property>
  </widget>
  <widget class="QLabel" name="marketing_label">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>210</y>
     <width>93</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Marketing:</string>
   </property>
  </widget>
  <widget class="QLabel" name="marketing_amount_label">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>210</y>
     <width>67</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buywood_button">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>390</y>
     <width>181</width>
     <height>42</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu</family>
     <pointsize>11</pointsize>
     <weight>7</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(143, 89, 2);
font: 57 11pt &quot;Ubuntu&quot;;
color: rgb(238, 238, 236);</string>
   </property>
   <property name="text">
    <string>Buy Wood
$50.00 / 100.00 m</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buygraphite_button">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>390</y>
     <width>181</width>
     <height>42</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu</family>
     <pointsize>11</pointsize>
     <weight>7</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(85, 87, 83);
font: 57 11pt &quot;Ubuntu&quot;;
color: rgb(238, 238, 236);</string>
   </property>
   <property name="text">
    <string>Buy Graphite
$50.00 / 100.00 m</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>210</width>
     <height>24</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="publicdemand_layout">
    <item>
     <widget class="QLabel" name="publicdemand_label">
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Public Demand:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="publicdemand_int">
      <property name="font">
       <font>
        <family>Ubuntu</family>
        <pointsize>12</pointsize>
        <weight>7</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 57 12pt &quot;Ubuntu&quot;;</string>
      </property>
      <property name="text">
       <string>x amount</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="main_title">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>10</y>
     <width>191</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu</family>
     <pointsize>20</pointsize>
     <weight>7</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 58 20pt &quot;Ubuntu&quot;;
</string>
   </property>
   <property name="text">
    <string>Pencil Producer</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>248</width>
     <height>24</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="pencilinventory_layout">
    <item>
     <widget class="QLabel" name="pencilinventory_label">
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Pencils in Inventory:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="pencilinventory_int">
      <property name="font">
       <font>
        <family>Ubuntu</family>
        <pointsize>12</pointsize>
        <weight>7</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 57 12pt &quot;Ubuntu&quot;;
</string>
      </property>
      <property name="text">
       <string>x amount</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>300</y>
     <width>181</width>
     <height>24</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="woodleft_layout">
    <property name="spacing">
     <number>9</number>
    </property>
    <item>
     <widget class="QLabel" name="wooleft_label">
      <property name="minimumSize">
       <size>
        <width>16</width>
        <height>0</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Wood Left:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="woodleft_int">
      <property name="font">
       <font>
        <family>Ubuntu</family>
        <pointsize>12</pointsize>
        <weight>7</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 57 12pt &quot;Ubuntu&quot;;</string>
      </property>
      <property name="text">
       <string>  x int</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>340</y>
     <width>181</width>
     <height>24</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="graphiteleft_layout">
    <item>
     <widget class="QLabel" name="graphiteleft_label">
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Graphite Left:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="graphiteleft_int">
      <property name="font">
       <font>
        <family>Ubuntu</family>
        <pointsize>12</pointsize>
        <weight>7</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 57 12pt &quot;Ubuntu&quot;;</string>
      </property>
      <property name="text">
       <string>  x int</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="pencilsproduced_label">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>71</y>
     <width>230</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Pencils produced (to date):</string>
   </property>
  </widget>
  <widget class="QLabel" name="pencilsproduced_int">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>72</y>
     <width>68</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu</family>
     <pointsize>12</pointsize>
     <weight>7</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 57 12pt &quot;Ubuntu&quot;;</string>
   </property>
   <property name="text">
    <string>x amount</string>
   </property>
  </widget>
  <widget class="QLabel" name="price_label">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>191</y>
     <width>127</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Price of Pencil:</string>
   </property>
  </widget>
  <widget class="QLabel" name="price_int">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>191</y>
     <width>47</width>
     <height>19</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu</family>
     <pointsize>12</pointsize>
     <weight>7</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 57 12pt &quot;Ubuntu&quot;;</string>
   </property>
   <property name="text">
    <string>x price</string>
   </property>
  </widget>
  <widget class="QPushButton" name="price_increase_button">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>220</y>
     <width>80</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sans Bold</family>
     <pointsize>25</pointsize>
     <weight>7</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(78, 154, 6);
font: 57 25pt &quot;Sans Bold&quot;;
color: rgb(0, 0, 0)</string>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="QPushButton" name="price_decrease_button">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>220</y>
     <width>81</width>
     <height>71</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>Sans Bold</family>
     <pointsize>45</pointsize>
     <weight>7</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(78, 154, 6);
font: 57 45pt &quot;Sans Bold&quot;;
color: rgb(0, 0, 0)</string>
   </property>
   <property name="text">
    <string>-</string>
   </property>
   <property name="iconSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>550</y>
     <width>921</width>
     <height>181</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../attendance-clipart-pencil-3.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
